client/app/project/project.html

```diff
@@ -1053,7 +1053,14 @@ <h6>{{ 'Options' | translate }}</h6>
                             </a>
                         </div>
                     </div>
-                    <div class="ol-control legend-control">
+                    <div ng-show="projectCtrl.projectData.annotations">
```

Comment:
Would it make sense to separate the general Task Annotation work and the one that addresses the specific application as an ML layer on the project page?